### PR TITLE
fix: Use of Spinner & Icon instead of custom components

### PR DESCRIPTION
**Pull request by GoOz**
- Use of `Spinner` component instead of custom `Loading` component
- Use of `Icon` component instead of embed icon

~~⚠️ wait for merge of https://github.com/cozy/cozy-ui/pull/971~~
~~⚠️ wait for merge of https://github.com/cozy/cozy-ui/pull/990~~

### CODE REVIEW

**Comment by GoOz** [comment]
[Fix Spinners on Home + Settings + Banks](https://trello.com/c/vC3pMfcd/1401-fix-spinners-on-home-settings-banks)

File: src/components/StorageView.jsx [hunk 1]
```diff
@@ -36,7 +36,9 @@ class StorageView extends Component {
         <h3 className={viewStyles['set-view-subtitle']}>
           {t('StorageView.storage_title')}
         </h3>
-        {isFetching && <Loading />}
```
Comment (CPatchane) [inline src/components/StorageView.jsx]: I don't see this Loading component being removed, we still needed it?

Comment (GoOz) [reply src/components/StorageView.jsx]: It has been removed, check again. :)

Comment (CPatchane) [reply src/components/StorageView.jsx]: OK my bad 👍

**Comment by GoOz** [comment]
Will be fixed when UI is updated.

**Pull request merged**